### PR TITLE
Speed up rate of sending commands.

### PR DESCRIPTION
**Pull request by jamessynge**
And continue sending commands a bit after first getting the desired
response, in case that helps with momentum.

### CODE REVIEW

**Comment by jamessynge** [comment]
Will test with real hardware later today after the snow showers are done over Wheaton.

File: pocs/dome/astrohaven.py [hunk 1]
```diff
@@ -157,7 +159,8 @@ def _full_move(self, send, target_feedback):
         # In other words, we'll try to read status, but if it isn't available,
         # we'll just send another command.
         saved_timeout = self.serial.ser.timeout
-        self.serial.ser.timeout = 0.2
+        self.serial.ser.timeout = max(0.0000001, AstrohavenDome.MOVE_LISTEN_TIMEOUT)
```
Comment (wtgee) [inline pocs/dome/astrohaven.py]: Just to guarantee timeout is never 0?

Comment (jamessynge) [reply pocs/dome/astrohaven.py]: Yes, though I now see that 0 does not imply forever. None is used for that. 0 means non-blocking (i.e. read as much as available AND requested, but don't wait if requested amount/terminator is not there).

**Comment by jamessynge** [comment]
FYI, still cloudy at Wheaton. Next clearing appears to be 10am tomorrow.

**Comment by jamessynge** [comment]
Worked great at Wheaton this afternoon.

**Pull request merged**